docs/UserGuide.md

```diff
@@ -19,26 +19,233 @@ To quickly get started with CLI-nton, follow these simple steps:
 CLI-nton offers a variety of features to streamline your task management process:
 
 
+Here are the sections for the User Guide covering the `next`, `prev`, `month`, `week`, and `quit` commands:
+
 ### Moving to Next Week or Month: `next`
 
-Moves to the next week or month view.
+The `next` command allows you to move to the next week or next month, depending on whether you are currently in the week or month view.
 
-Example of usage:
+**Usage:**
 
 ```
 next
 ```
 
+**Examples:**
+
+If you are currently in the **Week view:**
+```
+Week View: 31/03/2024 - 06/04/2024
```

Comment:
Perhaps could use pictures instead of typing out the output? Could feel "clunky" as a reader